src/plugins/vis_types/pie/public/components/visualization_noresults.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const VisualizationNoResults = ({ hasNegativeValues = false }) => {
+  return (
+    <EuiEmptyPrompt
+      iconType="visualizeApp"
+      iconColor="default"
+      data-test-subj="pieVisualizationError"
+      body={
+        <EuiText size="xs">
+          {hasNegativeValues
+            ? i18n.translate('visTypePie.negativeValuesFound', {
+                defaultMessage:
+                  "Pie charts can't render with negative values. Try a different visualization type.",
```

Comment:
If it's applicable to pie charts, it's applicable to all partition charts (treemap, donut, sunburst) so the word has to be either generalized (eg. "partition charts" or "part to whole charts") or the appropriate chart name should be inserted ("pie chart", "treemap", "mosaic plot"). Enlisting these for clarity, as eg. a treemap is almost never called a "treemap chart", it's just treemap.

Reply:
Visualize only supports pie and donut and we dont plan to support more types of partition charts. I changed it to `partition charts`